ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/MinaSshdManagementClient.java

```diff
@@ -200,6 +202,49 @@ public GenerateAndRegisterSshTunnelingKeyPairResponse generateAndRegisterSshTunn
         }
     }
 
+    /**
+     * Wraps call to unregisterSshTunnelingKey.
+     *
+     * @param requestId         the request ID for the request
+     * @param minaSshdServiceId the minasshd service ID
+     * @param keyId             the key ID
+     * @return the response
+     * @throws CcmException if an exception occurs
+     */
+    public UnregisterSshTunnelingKeyResponse unregisterSshTunnelingKey(
+            String requestId, String minaSshdServiceId, String keyId) throws CcmException {
+        checkNotNull(requestId);
+        checkNotNull(minaSshdServiceId);
+        checkNotNull(keyId);
+
+        MinaSshdManagementBlockingStub blockingStub = newStub(requestId);
+        UnregisterSshTunnelingKeyRequest.Builder requestBuilder = UnregisterSshTunnelingKeyRequest.newBuilder()
+                .setMinaSshdServiceId(minaSshdServiceId)
+                .setKeyId(keyId);
+
+        try {
+            if (LOGGER.isDebugEnabled()) {
```

Comment:
it doesn't look like you are doing any work in this block so the `LOGGER.isDebugEnabled` check is unnecessary

Reply:
Done.